### DISCUSSION TITLE
So i have issues monika cannot read the gifts i give to her i already update it but she still cant read the file pls comment in down below or chat me in dis/discord here : #2214

### DISCUSSION THREAD

**Discussion by LILSALSA123** [Monika-After-Story/MonikaModDev#9194] [QA - tech support + general]
i alrady update or paste the mod asset in the game but it still wont work and yes i download the right spritepacks pls help me! (SOMEONE SAVE MY LIFE)

**Comment by Justformas** [comment]
Try following this guide: https://github.com/Monika-After-Story/MonikaModDev/wiki/Gift-Giving-Guide

Also make sure your spritepacks are installed correctly: https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Sprite-Objects#installing-sprite-packs

**Comment by Booplicate** [comment]
> Lastly you need at least 30 affection points to give her gifts from spritepacks

You can always gift them to her, you can't ask to wear them.

**Reply by Justformas** [reply]
> > Lastly you need at least 30 affection points to give her gifts from spritepacks
> 
> You can always gift them to her, you can't ask to wear them.

Oh I see now, going back to when you said that under the context. Though I couldn't seem to find any Good/Happy/30 affection requirement for spritepacks in any guides here. The closest thing I found was that when Monika reaches "Love" she can change her clothes to ones you have seen her wearing, but for spritepacks you gain that option much sooner.
